src/closure-defined-fs-annotation.js

```diff
@@ -1,29 +0,0 @@
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var Browser;
```

Comment:
`Browser` annotation is now handled in `library_fs.js`. The other annotations below do not seem to have a test case in the suite, or the annotations have become redundant over time.